src/openforms/services/api/viewsets.py

```diff
@@ -20,7 +20,7 @@
     )
 )
 @mark_experimental
-class ServiceViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
+class ServiceViewSet(viewsets.ReadOnlyModelViewSet):
```

Comment:
I think it does the same as having `mixins.ListModelMixin, viewsets.GenericViewSet`, except it also adds `mixins.RetrieveModelMixin` (which is required for hyperlinked serializers)

Reply:
for completeness sake - `viewsets.GenericViewSet` does not expose any actions at all, so before this would only expose the `list` action. `ReadOnlyModelViewSet` exposes the `list` and `retrieve` actions (through their respective mixins).